src/SAREhub/Client/Processor/LogProcessor.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+
+namespace SAREhub\Client\Processor;
+
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+use SAREhub\Client\Message\Exchange;
+use SAREhub\Client\Util\IdAware;
+
+class LogProcessor implements Processor, LoggerAwareInterface, IdAware
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    private $id;
+
+    private $logLevel = "debug";
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function setLogLevel(string $logLevel)
+    {
+        $this->logLevel = $logLevel;
```

Comment:
check level is valid with LogLevel class constants

Reply:
ready